internal/service/encryptionatrestprivateendpoint/resource_test.go

```diff
@@ -70,6 +71,53 @@ func basicTestCase(tb testing.TB) *resource.TestCase {
 	}
 }
 
+func TestAccEncryptionAtRestPrivateEndpoint_approveEndpointWithAzureProvider(t *testing.T) {
+	acc.SkipTestForCI(t) // uses azure/azapi Terraform provider which can log sensitive information in CI like Azure subscriptionID used in parent_id of the resource
+
+	var (
+		subscriptionID    = os.Getenv("AZURE_SUBSCRIPTION_ID")
+		resourceGroupName = os.Getenv("AZURE_RESOURCE_GROUP_NAME")
+		projectID         = os.Getenv("MONGODB_ATLAS_PROJECT_EAR_PE_ID")
+		keyVaultName      = os.Getenv("AZURE_KEY_VAULT_NAME")
+		azureKeyVault     = &admin.AzureKeyVault{
+			Enabled:                  conversion.Pointer(true),
+			RequirePrivateNetworking: conversion.Pointer(true),
+			AzureEnvironment:         conversion.StringPtr("AZURE"),
+			ClientID:                 conversion.StringPtr(os.Getenv("AZURE_CLIENT_ID")),
+			SubscriptionID:           conversion.StringPtr(subscriptionID),
+			ResourceGroupName:        conversion.StringPtr(resourceGroupName),
+			KeyVaultName:             conversion.StringPtr(keyVaultName),
+			KeyIdentifier:            conversion.StringPtr(os.Getenv("AZURE_KEY_IDENTIFIER")),
+			Secret:                   conversion.StringPtr(os.Getenv("AZURE_APP_SECRET")),
+			TenantID:                 conversion.StringPtr(os.Getenv("AZURE_TENANT_ID")),
+		}
+		region = os.Getenv("AZURE_PRIVATE_ENDPOINT_REGION")
+	)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acc.PreCheckBasic(t); acc.PreCheckEncryptionAtRestEnvAzure(t); acc.PreCheckPreviewFlag(t) },
+		ExternalProviders:        acc.ExternalProvidersOnlyAzapi(),
+		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
+		CheckDestroy:             checkDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: configPrivateEndpointAzureBasic(projectID, azureKeyVault, region, false),
```

Comment:
knit: similar comment, rename `configPrivateEndpointAzureBasic`, `checkPrivateEndpointAzureBasic` to something like `configAzureBasic`, `checkAzureBasic`...

Reply:
updated